pkg/kubelet/dockertools/manager.go

```diff
@@ -231,6 +237,7 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 		enableCustomMetrics:    enableCustomMetrics,
 		configureHairpinMode:   hairpinMode,
+		versionCache:           cache.NewVersionCache(),
```

Comment:
I expected version cache to take in an updater function
```go
versionCache: cache.NewVersionCache(func() (string, string, error) {
return dm.getVersionInfo()
})
```

Version cache should be invoking that update method periodically in the background.